Can not compile 9.3.3 on Debian

Hello,

for the record, It does not compile on Debian.

```
zmqeventsupplier.cpp:689:18: error: cannot convert 'zmq::socket_t' to 'void*'
  689 |     if (zmq_bind(*(ms.pub_socket),ms.endpoint.c_str()) != 0)
      |                  ^~~~~~~~~~~~~~~~
      |                  |
      |                  zmq::socket_t
In file included from /usr/include/zmq.hpp:38,
                 from ../../../lib/cpp/client/event.h:41,
                 from ./tango.h:146,
                 from zmqeventsupplier.cpp:33:
/usr/include/zmq.h:485:32: note:   initializing argument 1 of 'int zmq_bind(void*, const char*)'
  485 | ZMQ_EXPORT int zmq_bind (void *s_, const char *addr_);
      |                          ~~~~~~^~
```

you can find the full log here

https://salsa.debian.org/science-team/tango/-/jobs/290586

cheers

--- a/README.md
+++ b/README.md
@@ -32,7 +32,7 @@ package:
   [copydir] DEPRECATED - The copydir task is deprecated.  Use copy instead.
   [copydir] Copying 710 files to /home/travis/build/tango-controls/TangoSourceDistribution/build/package/tango-9.3.2
       [tar] Building tar: /home/travis/build/tango-controls/TangoSourceDistribution/build/tango-master.tar.gz
-$> 
+$>
 ```
 
 `ant build` creates _build_ folder where it performs all required manipulations to fetch, download and assemble all the components of Tango Source Distribution: c++ from githug repos; java from maven repos. Result is placed into _build/distr_

--- a/assets/README
+++ b/assets/README
@@ -1,4 +1,4 @@
-		TANGO source code release version 9.3.3 - README (08/2019)
+		TANGO source code release version 9.3.4 - README (11/2019)
 		==========================================================
 
 
@@ -26,7 +26,7 @@ This is the README for the TANGO source code release. TANGO is a
 toolkit for building object oriented control systems based on CORBA ans ZMQ. 
 TANGO is a joint effort of the several European research institutes.
 
-This source code release is based on the latest version of TANGO 9 (9.3.3)
+This source code release is based on the latest version of TANGO 9 (9.3.4)
 and is intended for Unix platforms. A binary version for Windows of TANGO 
 is available. The source code contained in this package 
 runs on Windows but the Makefile don't.
@@ -230,7 +230,7 @@ ready to be installed.
 Example (using mysql):								
         
 		Source code location:   ../..
-        Version:                9.3.3
+        Version:                9.3.4
         Compiler:               gcc,g++
 
         OMNIORB PATH:           /segfs/tango/ORB/omniORB4.2.1/ubuntu1404
@@ -439,7 +439,7 @@ as yes at the end of the configure script execution)
 7-1 - UPDATING FROM A TANGO 9.2.5 DATABASE
 --------------------------------------
 
-Tango 9.3.3 includes an update of the database:
+Tango 9.3.4 includes an update of the database:
 (1) new DServer always allowed command: EventConfirmSubscription,
 (2) new entry for TangoRestServer in device table.
 
@@ -452,7 +452,7 @@ b - Backup your database (Recommended, not mandatory)
 c - Cd to the <install_dir>/share/tango/db directory
 
 d - Run the update script:
-	mysql -u[user] -p[password] < ./update_db_from_9.2.5_to_9.3.3.sql
+	mysql -u[user] -p[password] < ./update_db_from_9.2.5_to_9.3.4.sql
 
 e - Restart your database server(s)
 
@@ -460,7 +460,7 @@ e - Restart your database server(s)
 7-2 - UPDATING FROM A TANGO 9 (< 9.2.5) DATABASE
 --------------------------------------
 
-Tango 9.3.3 includes an update of the database:
+Tango 9.3.4 includes an update of the database:
 (1) new DServer always allowed command: EventConfirmSubscription,
 (2) new entry for TangoRestServer in device table.
 (3) update of the stored procedure (release 1.13).
@@ -477,14 +477,14 @@ d - Run the update script to update the stored procedure:
 	./update_db.sh
 	
 e - Run the update script for the new entries (1) and (2) in the DB:
-	mysql -u[user] -p[password] < ./update_db_from_9.2.5_to_9.3.3.sql
+	mysql -u[user] -p[password] < ./update_db_from_9.2.5_to_9.3.4.sql
 	
 f - Restart your database server(s)
 
 7-3 - UPDATING FROM A TANGO 8 DATABASE
 --------------------------------------
 
-Tango 9.3.3 includes an update of the database:
+Tango 9.3.4 includes an update of the database:
 (1) new DServer always allowed command: EventConfirmSubscription.
 (2) new entry for TangoRestServer in device table.
 (3) update of the stored procedure (release 1.13).
@@ -500,7 +500,7 @@ b - Backup your database (Recommended, not mandatory)
 c - Cd to the <install_dir>/share/tango/db directory
 
 d - Run the update script:
-	mysql -u[user] -p[password] < ./update_db_from_8_to_9.3.3.sql
+	mysql -u[user] -p[password] < ./update_db_from_8_to_9.3.4.sql
 	
 e - Restart your database server(s)
 
@@ -524,7 +524,7 @@ b - Backup your database (Recommended, not mandatory)
 c - Cd to the <install_dir>/share/tango/db directory
 
 d - Run the update script:
-	mysql -u[user] -p[password] < ./update_db_from_7_to_9.3.3.sql
+	mysql -u[user] -p[password] < ./update_db_from_7_to_9.3.4.sql
 	
 e - Restart your database server(s)
 

--- a/assets/configure.ac
+++ b/assets/configure.ac
@@ -501,9 +501,9 @@ AC_OUTPUT(Makefile
 	  cppserver/database/create_db_tables.sql
 	  cppserver/database/update_db.sh
 	  cppserver/database/update_db.sql
-	  cppserver/database/update_db_from_8_to_9.3.3.sql
-	  cppserver/database/update_db_from_7_to_9.3.3.sql
-	  cppserver/database/update_db_from_9.2.5_to_9.3.3.sql
+	  cppserver/database/update_db_from_8_to_9.3.4.sql
+	  cppserver/database/update_db_from_7_to_9.3.4.sql
+	  cppserver/database/update_db_from_9.2.5_to_9.3.4.sql
       cppserver/database/rem_history.sql
 	  cppserver/starter/Makefile
   	  cppserver/tangotest/Makefile

--- a/assets/cppserver/database/Makefile.am
+++ b/assets/cppserver/database/Makefile.am
@@ -38,9 +38,9 @@ db_DATA = \
     stored_proc.sql \
     update_db.sh \
     update_db.sql \
-    update_db_from_8_to_9.3.3.sql \
-    update_db_from_7_to_9.3.3.sql \
-    update_db_from_9.2.5_to_9.3.3.sql \
+    update_db_from_8_to_9.3.4.sql \
+    update_db_from_7_to_9.3.4.sql \
+    update_db_from_9.2.5_to_9.3.4.sql \
     rem_history.sql
 
 ## This is to make sure that the create-db script is run on each make all.
@@ -56,9 +56,9 @@ EXTRA_DIST = \
     create_db_tables.sql.in \
     stored_proc.sql.in \
     update_db.sh.in \
-    update_db_from_8_to_9.3.3.sql.in \
-    update_db_from_7_to_9.3.3.sql.in \
-    update_db_from_9.2.5_to_9.3.3.sql.in \
+    update_db_from_8_to_9.3.4.sql.in \
+    update_db_from_7_to_9.3.4.sql.in \
+    update_db_from_9.2.5_to_9.3.4.sql.in \
     update_db.sql.in \
     rem_history.sql.in
 

--- a/cleanup_distrib
+++ b/cleanup_distrib
@@ -19,8 +19,8 @@ rm -Rf cppserver/database/winnt
 rm -Rf cppserver/database/win64
 rm -f cppserver/database/Makefile
 rm -f cppserver/database/README
-rm -f cppserver/database/update_db_from_5_to_9.3.3.sql.in
-rm -f cppserver/database/update_db_from_6_to_9.3.3.sql.in
+rm -f cppserver/database/update_db_from_5_to_9.3.4.sql.in
+rm -f cppserver/database/update_db_from_6_to_9.3.4.sql.in
 
 rm -Rf cppserver/starter/doc_html
 rm -f cppserver/starter/Makefile

--- a/distribution.properties
+++ b/distribution.properties
@@ -8,9 +8,9 @@ rest-server-root-repo=https://github.com/tango-controls/rest-server/releases
 #current = tango lib major + $age
 #revision = tango lib patch
 #age = tango lib minor
-version-info=12:3:3
+version-info=12:3:4
 #lib
-cppTango=9.3.3
+cppTango=9.3.4-rc1
 JTango-ver=9.5.17
 tango-idl=5.0.1
 jive-ver=7.22
@@ -26,10 +26,10 @@ rest-server-ver=1.14
 #tool_panels=
 #cppserver
 TangoTest=TangoTest-Release-2.1
-TangoDatabase=Database-Release-5.12
+TangoDatabase=Database-Release-5.13
 TangoAccessControl=TangoAccessControl-Release-2.14
 starter=Starter-7.2
 #utils
 tango_admin=Release_1.14
 #mtangorest.server=rc4-2.9
-tango-doc=9.3.3
+tango-doc=9.3.4

